metat_tasks.wdl

```diff
@@ -0,0 +1,665 @@
+      File count_table
+      File? count_ig
+      File? count_log 
+      File readcount_info 
+      # output tables
+      File gff_json
+      File rc_json 
+      File gff_rc_json 
+      File cds_json 
+      File sense_json 
+      File anti_json 
+      File top100_json 
+      File sorted_json 
+      File sorted_tsv 
+   }
+
+   command <<<
+      set -eou pipefail
+      mkdir -p ~{qcdir}
+      mkdir -p ~{assemdir}
+      mkdir -p ~{readmap}
+      mkdir -p ~{annodir}
+      end=`date --iso-8601=seconds`
+
+      #move qc objects
+      cp ~{filtered} ~{qcdir}/~{prefix}_filtered.fastq.gz
+      cp ~{filtered_stats} ~{qcdir}/~{prefix}_filterStats.txt
+      cp ~{filtered_stats2} ~{qcdir}/~{prefix}_filterStats2.txt	
+      cp ~{rqc_info} ~{qcdir}/~{prefix}_rqc.info	
+      # Generate QC objects
+      /scripts/rqcstats.py ~{filtered_stats} > stats.json
+      cp stats.json ~{qcdir}/~{prefix}_qc_stats.json 
+
+      # move assy objects
+      cp ~{tar_bam} ~{assemdir}/~{prefix}_bamfiles.tar
+      cp ~{contigs} ~{assemdir}/~{prefix}_contigs.fna
+      cp ~{scaffolds} ~{assemdir}/~{prefix}_scaffolds.fna
+      cp ~{readlen} ~{assemdir}/~{prefix}_readlen.txt
+      cp ~{sam} ~{assemdir}/~{prefix}_pairedMapped.sam.gz
+      cp ~{bam} ~{assemdir}/~{prefix}_pairedMapped.bam.gz
+      cp ~{asmstats} ~{assemdir}/~{prefix}_scaffold_stats.json 
+      cp ~{asse_info} ~{assemdir}/~{prefix}_assy.info 
+      cp ~{asse_log} ~{assemdir}/~{prefix}_spades.log
+
+      # move anno objects
+      cp ~{proteins_faa} ~{annodir}/~{prefix}_proteins.faa
+      cp ~{structural_gff} ~{annodir}/~{prefix}_structural_annotation.gff
+      cp ~{ko_ec_gff} ~{annodir}/~{prefix}_ko_ec.gff
+      cp ~{gene_phylogeny_tsv} ~{annodir}/~{prefix}_gene_phylogeny.tsv
+      cp ~{functional_gff} ~{annodir}/~{prefix}_functional_annotation.gff
+      cp ~{ko_tsv} ~{annodir}/~{prefix}_ko.tsv
+      cp ~{ec_tsv} ~{annodir}/~{prefix}_ec.tsv
+      cp ~{lineage_tsv} ~{annodir}/~{prefix}_scaffold_lineage.tsv
+      cp ~{stats_tsv} ~{annodir}/~{prefix}_anno_stats.tsv
+      cp ~{cog_gff} ~{annodir}/~{prefix}_cog.gff
+      cp ~{pfam_gff} ~{annodir}/~{prefix}_pfam.gff
+      cp ~{tigrfam_gff} ~{annodir}/~{prefix}_tigrfam.gff
+      cp ~{smart_gff} ~{annodir}/~{prefix}_smart.gff
+      cp ~{supfam_gff} ~{annodir}/~{prefix}_supfam.gff
+      cp ~{cath_funfam_gff} ~{annodir}/~{prefix}_cath_funfam.gff
+      cp ~{crt_gff} ~{annodir}/~{prefix}_crt.gff
+      cp ~{genemark_gff} ~{annodir}/~{prefix}_genemark.gff
+      cp ~{prodigal_gff} ~{annodir}/~{prefix}_prodigal.gff
+      cp ~{trna_gff} ~{annodir}/~{prefix}_trna.gff
+      cp ~{rfam_gff} ~{annodir}/~{prefix}_rfam.gff
+      cp ~{product_names_tsv} ~{annodir}/~{prefix}_product_names.tsv
+      cp ~{crt_crisprs} ~{annodir}/~{prefix}_crt.crisprs
+      cp ~{imgap_version} ~{annodir}/~{prefix}_anno.info
+      cp ~{renamed_fasta} ~{annodir}/~{prefix}_renamed_contigs.fna
+      cp ~{map_file} ~{annodir}/~{prefix}_map.fna
```

Comment:
This file is incorrectly named nmdc_xxxxxxx_map.fna, this should be _contig_names_mapping.tsv